hindi/Bhagat_Singh/bhagat_singh.md

```diff
@@ -0,0 +1,44 @@
+Bhagat Singh aka Shaheed Bhagat Singh (28 September 1907 – 23 March 1931) was an 
+[Indian](https://simple.wikipedia.org/wiki/India) socialist and a revolutionary. 
+He is considered to be one of the most influential revolutionaries of the 
+[Indian Independence Movement](https://simple.wikipedia.org/wiki/Indian_independence_movement).
+
+He was born in a [Sikh](https://simple.wikipedia.org/wiki/Sikh) family. 
+His family had earlier been involved in revolutionary activities against the 
+[British Raj](https://simple.wikipedia.org/wiki/Sikh). 
+When Bhagat Singh was a teenager, he studied European revolutionary movements.
+He became attracted to [anarchist](https://simple.wikipedia.org/wiki/Anarchist) and 
+[Marxist](https://simple.wikipedia.org/wiki/Marxist) ideologies.
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.